packages/editor-ui/src/api/workflow-webhooks.ts

```diff
@@ -7,25 +7,25 @@ const CONTACT_EMAIL_SUBMISSION_ENDPOINT = '/accounts/onboarding';
 
 export async function fetchNextOnboardingPrompt(
 	instanceId: string,
-	currentUser: IUser,
+	currentUser: IUser | null,
```

Comment:
In which case can this be `null`?

Reply:
It can be `null` in the store. But I guess this is not something that we should worry about at this place. Will refactor so it's not called if user is `null`

Reply:
This shouldn't happen by the time this function is called but still, from the store's perspective, there is a point where user is `null`